<launch>

  <node pkg="joy" type="joy_node" name="joy_node" output="screen">
    <param name="dev" value="/dev/input/js0" />
    <param name="deadzone" value="0.1" />
  </node>

  <node pkg="joy_custom" name="joy_controller_node" type="joy_controller_node">
    <rosparam command="load" file="$(find joy_custom)/config/JoyCustom.config.yaml" />
    <!-- <remap from="cmd_vel" to="/whill/controller/cmd_vel" />-->
  </node>
  
</launch>

